ion-schema/src/ion_schema_version.rs

```diff
@@ -8,11 +8,12 @@ mod private {
 }
 
 /// Trait for ISL version marker types.
-pub trait IslVersion: private::IslVersion {
+pub trait IslVersion: private::IslVersion + Clone {
```

Comment:
🗺️ For some reason, `PhantomData<T>` doesn't implement `Clone` unless `T` implements `Clone`, even though `PhantomData` is a zero-sized type (so it should always be trivially `Copy` and `Clone`). So now we have a `Clone` requirement for `IslVersion`.